model-optimizer/extensions/front/tf/ObjectDetectionAPI.py

```diff
@@ -486,6 +486,22 @@ def nodes_to_remove(self, graph: Graph, match: SubgraphMatch):
                 new_nodes_to_remove.remove(node_to_keep)
         return new_nodes_to_remove
 
+    def is_preprocessing_applied_before_resize(self, to_float: Node, mul: Node, sub: Node):
+        """
+        The function checks if the output of 'to_float' operation which is consumed by 'mul' or 'sub'. If this is true
+        that means that the pre-processing (mean/scale) is applied before the image resize. The image resize was applied
+        first in the original version of the TF OD API models, but in the recent versions it is applied after.
+
+        :param to_float: the Cast node which converts the input tensor to Float
+        :param mul: the Mul node (can be None)
+        :param sub: the Sub node
+        :return: the result of the check
+        """
+        if mul is not None:
+            return any([port.node.id == mul.id for port in to_float.out_port(0).get_destinations()])
+        else:
+            return any([port.node.id == sub.id for port in to_float.out_port(0).get_destinations()])
```

Comment:
I think that assert for `sub`  that it is not None is needed

Reply:
Fixed.